interface/ten_ai_base/tts2.py

```
@@ -114,6 +115,9 @@ async def on_data(self, ten_env: AsyncTenEnv, data: Data) -> None:
                 f"on_data tts_text_input:  {json.dumps(json.loads(data_payload), ensure_ascii=False)}",
                 category=LOG_CATEGORY_KEY_POINT,
             )
+            # Update request_id and store metadata when a new request arrives
+            if not t.request_id in self.metadatas:
```

Ruff (0.14.0)

Test for membership should be `not in`

Convert to `not in`

(E713)